cfme/common/provider.py

```diff
@@ -310,7 +310,7 @@ def validate_stats(self, ui=False):
                      {'ui': ui},
                      message="do_stats_match_db",
                      num_sec=1000,
-                     delay=60)
+                     delay=120)
```

Comment:
@psav No, it's actually the right function. Here is what Erez says: "Before: We only collected the images from the pods
Now: We collect the images from the pods AND also from Openshift. Images from Openshift have more information to them. If a pod is running an image that we can link to an Openshift image and we unite them."
So it's pulling images from Openshift but there are also additional images that are being added. The problem is. the validation we are doing right now is using this function:
    def list_image(self):
        """Returns list of images (derived from pods)"""
        entities = []
        entities_j = self.api.get('pod')[1]['items']
        for entity_j in entities_j:
            imgs_j = entity_j['status'].get('containerStatuses', [])
            for img_j in imgs_j:
                _, name, tag = self._parse_image_info(img_j['image'])
                img = Image(name, tag, img_j['imageID'])
                if img not in entities:
                    entities.append(img)
        return entities

But it only pulls the Openshift images, while ignoring the other ones. In CFME 5.6, it was just the Openshift images.